src/commands/listIsActive.ts

```diff
@@ -1,10 +1,13 @@
 import { EditorState } from "prosemirror-state"
 import { NodeType, Schema } from "prosemirror-model"
-import { hasParentNodeOfType } from "prosemirror-utils"
+import { findParentNodeOfType } from "prosemirror-utils"
 
 export function listIsActive(
   state: EditorState,
-  type: NodeType<Schema<any, any>>
+  type: NodeType<Schema<any, any>>,
+  listNodes: typeof type[]
```

Comment:
Passing schema for list types from the RTE, as the schema varies compared to the one in `prosemirror`.
The usage of this method will look like https://github.com/cultureamp/kaizen-design-system/pull/2909/files#diff-2b0b59d88def8c4b25c53c18f2487ce79955a98fe86ee4a82847ff0b45b370e3R252

Reply:
Nice, I think that will work well with the current implementation